src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double height;
+    private double aBase;
+    private double bBase;
+
+    public IsoscelesTrapezoid(double height, double aBase, double bBase, String color) {
+        super(color);
+        this.height = height;
+        this.aBase = aBase;
+        this.bBase = bBase;
+    }
+
+    public double getbBase() {
+        return bBase;
+    }
+
+    public void setbBase(double bBase) {
+        this.bBase = bBase;
+    }
```

Comment:
Method names should follow the camelCase naming convention. So, 'getbBase' should be 'getBBase', and 'setbBase' should be 'setBBase'.